tests/ConstructAndDestroyComponentTests.js

```diff
@@ -0,0 +1,72 @@
+"use strict";
+
+var fluid = require("infusion"),
+    gpii = fluid.registerNamespace("gpii"),
+    kettle = fluid.registerNamespace("kettle"),
+    path = require("path"),
+    configPath = path.resolve(__dirname, "configs");
+
+require("kettle");
+require("../src/Nexus.js");
+require("../src/test/NexusTestUtils.js");
+
+kettle.loadTestingSupport();
+
+fluid.registerNamespace("gpii.tests.nexus.constructComponent");
+
+gpii.tests.nexus.constructComponent.componentOptions = {
+    type: "fluid.modelComponent",
+    model: {
+        someModelPath: 2
+    }
+};
+
+gpii.tests.nexus.constructComponent.testDefs = [
+    {
+        name: "Construct and Destroy Component",
+        gradeNames: "gpii.test.nexus.testCaseHolder",
+        expect: 6,
+        config: {
+            configName: "gpii.tests.nexus.config",
+            configPath: configPath
+        },
+        testComponentPath: "nexusConstructComponentTestComponent",
```

Comment:
I had a go at constructing a component at a path containing a period and a forward slash. I'm seeing at error from FluidIoC.js. To support component paths with periods and slashes in the Nexus, I think we'll need to make some Infusion changes.

I've filed a JIRA issue: https://issues.gpii.net/browse/GPII-1701